dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -168,7 +168,7 @@ const decideImageWidths = ({
  * @see https://github.com/guardian/fastly-image-service/blob/main/fastly-io_guim_co_uk/src/main/resources/varnish/main.vcl
  *
  */
-const generateImageURL = ({
+export const generateImageURL = ({
```

Comment:
Maybe not for this PR but we could probably look at ways to refactor how we share code here